pkg/util/grpcclientutil/grpcclientutil.go

```diff
@@ -63,7 +63,7 @@ func getInterfaceServiceClient() (InterfaceServiceClient, *grpc.ClientConn, cont
 		return nil, nil, nil, nil, err
 	}
 	client := NewInterfaceServiceClient(conn)
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*3)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*8)
```

Comment:
Can we please add this to constants file as CONTEXT_TIMEOUT and reference it here? 
Also, does it make sense to take a larger number (perhaps 30s)?

Reply:
Does it make sense for this call to keep retry?